README.md

```diff
@@ -16,40 +16,94 @@ Some of these are tricky!  Don't trust your first instinct.
 
 
 **a)** `"1" == 1`
+<pre>
+True. ==  tries to match types 
+</pre>
 
-**b)** `"1" === 1`
+**b)** `"1" === 1` 
+<pre> 
+False, not an exact match.
+</pre>
 
 **c)** `x == 'x'`
+<pre> 
+True, types converted. 
```

Comment:
Whether this is true or false actually depends on the value of `x`: if `var x = 'x'`, then it will be true, otherwise it will be false.